packages/server/tests/acceptance/rpc_batch3.spec.ts

```diff
@@ -49,9 +51,67 @@ import EstimateGasContract from '../contracts/EstimateGasContract.json';
 // Helper functions/constants from local resources
 import { EthImpl } from '../../../../packages/relay/src/lib/eth';
 import { predefined } from '../../../../packages/relay';
+import exp from 'constants';
+import e from 'express';
 
 chai.use(chaiExclude);
 
+const sendJsonRpcRequestWithDelay = (
```

Comment:
I'd suggest moving this method to helpers/utils.js or somewhere else to keep the batch 3 file clean. Also remove unsed import statements.

Reply:
Done.